### PR TITLE
Discovery fixes

### PR DESCRIPTION
**Pull request by seanyesmunt**
### Changes
- Add tags to publish page
  - Got tired of working with the `PublishForm` component so I broke it into a bunch of small components
- Moved to reach/tooltip for better tooltips (and added one for channels)
  - tooltip behavior is hard
- General fixes

### Notes
- Depends on https://github.com/lbryio/components/pull/11

### CODE REVIEW

File: src/ui/component/claimList/view.jsx [hunk 1]
```diff
@@ -36,10 +36,10 @@ export default function ClaimList(props: Props) {
   return (
     <section className={classnames('file-list')}>
       {header !== false && (
-        <div className={classnames('file-list__header', { 'file-list__header--small': type === 'small' })}>
+        <div className={classnames('claim-list__header', { 'claim-list__header--small': type === 'small' })}>
```
Comment (kauffj) [inline src/ui/component/claimList/view.jsx]: good refactor

File: src/ui/component/publishFile/view.jsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+      <header className="card__header">
+        <h2 className="card__title card__title--flex-between">{isStillEditing ? __('Edit') : __('Publish')}</h2>
+        {isStillEditing && <p className="card__subtitle">{__('You are currently editing a claim.')}</p>}
+      </header>
+
+      <div className="card__content">
+        <FileSelector currentPath={filePath} onFileChosen={handleFileChange} />
+        {!!isStillEditing && name && (
+          <p className="help">
+            {__("If you don't choose a file, the file from your existing claim")}
```
Comment (kauffj) [inline src/ui/component/publishFile/view.jsx]: When you notice split i18n strings like this, can you add an annotation of some kind? (I propose `//@i18nfixme`)

File: src/ui/component/publishFormErrors/view.jsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  return (
+    <div className="card__content error-text">
+      {!title && <div>{__('A title is required')}</div>}
+      {!name && <div>{__('A URL is required')}</div>}
+      {!bid && <div>{__('A deposit amount is required')}</div>}
+      {uploadThumbnailStatus === THUMBNAIL_STATUSES.IN_PROGRESS && (
+        <div>{__('Please wait for thumbnail to finish uploading')}</div>
+      )}
+      {!!editingURI && !isStillEditing && !filePath && (
+        <div>{__('You need to reselect a file after changing the LBRY URL')}</div>
```
Comment (kauffj) [inline src/ui/component/publishFormErrors/view.jsx]: Can this be re-written to say "Please"?

(also, why is this necessary?)

File: src/ui/component/publishName/bid-help-text.jsx [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+  let bidHelpText;
+
+  if (uri) {
+    if (isResolvingUri) {
+      bidHelpText = __('Checking the winning claim amount...');
+    } else if (!amountNeededForTakeover) {
+      bidHelpText = __('Any amount will give you the winning bid.');
+    } else {
+      bidHelpText = `${__('If you bid more than')} ${amountNeededForTakeover} LBC, ${__(
```
Comment (kauffj) [inline src/ui/component/publishName/bid-help-text.jsx]: `@i18nfixme`

(also, these split strings are basically not worth even calling the function on, but should be tagged either way)

File: src/ui/component/publishName/view.jsx [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+      prepareEdit(myClaimForUri, uri);
+    }
+  }
+
+  useEffect(() => {
+    let nameError;
+    if (!name) {
+      nameError = __('A name is required');
+    } else if (!isNameValid(name, false)) {
+      nameError = __('LBRY names cannot contain that symbol ($, #, @)');
```
Comment (kauffj) [inline src/ui/component/publishName/view.jsx]: Some symbols are missing here. It's probably worth adding a `RESERVED_CHARACTER` const to `lbryUri.js`.

File: src/ui/component/selectThumbnail/view.jsx [hunk 6]
```diff
@@ -160,6 +161,18 @@ class SelectThumbnail extends React.PureComponent<Props, State> {
         )}
 
         {status === THUMBNAIL_STATUSES.IN_PROGRESS && <p>{__('Uploading thumbnail')}...</p>}
+        <p className="help">
+          {(status === undefined && __('You should reselect your file to choose a thumbnail')) ||
+            (status === THUMBNAIL_STATUSES.API_DOWN ? (
+              __('Enter a URL for your thumbnail.')
+            ) : (
+              <React.Fragment>
+                {__('Upload your thumbnail (.png/.jpg/.jpeg/.gif) to')}{' '}
```
Comment (kauffj) [inline src/ui/component/selectThumbnail/view.jsx]: I do not think the file types need to be mentioned here. Not a strong opinion, though.

File: src/ui/component/publishFormErrors/view.jsx (hunk 3 above)
Comment (seanyesmunt) [reply src/ui/component/publishFormErrors/view.jsx]: It shouldn't be, but we aren't currently tracking the previous claim, after you change the name, because that could be another claim.
